vyper/parser/stmt.py

```diff
@@ -606,24 +602,21 @@ def _get_target(self, target):
                 raise_exception = True
```

Comment:
`list_name = target.value.id` above is no longer in use

Reply:
https://lgtm.com/projects/g/vyperlang/vyper/rev/pr-d5ea08f0db8196c51f8ab23dbe142eb2f58a1db5

Reply:
ooops